server/s3server.cc

```diff
@@ -1095,9 +1096,9 @@ int main(int argc, char **argv) {
          global_instance_list_index_layout.pver.f_key,
          global_instance_list_index_layout.layout_type);
 
-  // bucket_object_count_index_layout - will hold accountid/bucket_name as key,
+  // bucket_data_usage_index_layout - will hold accountid/bucket_name as key,
   // bucket medata as value.
-  rc = create_global_index(bucket_object_count_index_layout,
+  rc = create_global_index(bucket_data_usage_index_layout,
                            BUCKET_OBJECT_COUNT_INDEX_OID_U_LO);
```

Comment:
Rename `BUCKET_OBJECT_COUNT_INDEX_OID_U_LO` to `DATA_USAGE_ACCOUNTS_INDEX_OID_U_LO`.

Reply:
Done.